packages/synapse-react-client/src/utils/hooks/useLogin.test.tsx

```diff
@@ -46,6 +46,22 @@ function renderUseLogin(
   })
 }
 
+async function verify2FAResetStarted(
+  result: ReturnType<typeof renderUseLogin>['result'],
+  resetEndpoint: string,
+) {
+  await waitFor(() => {
+    expect(mockReset2FA).toHaveBeenCalledWith({
+      userId: twoFactorAuthErrorResponse.userId,
+      twoFaToken: twoFactorAuthErrorResponse.twoFaToken,
+      twoFaResetEndpoint: resetEndpoint,
+    })
+    expect(mockSetAccessTokenCookie).not.toHaveBeenCalled()
+    expect(onSuccessfulLoginFn).not.toHaveBeenCalled()
+    expect(result.current.step).toBe('DISABLE_2FA_PROMPT')
```

Comment:
This set of assertions was extracted. I added an assertion `expect(result.current.step).toBe('DISABLE_2FA_PROMPT')` which fails when it encounters this bug.